content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
Ok. My only question is whether we still need to acknowledge National Guard so that they don't mistakenly try to apply through our form.